hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/processors/CustomMessageCallProcessor.java

```diff
@@ -124,12 +123,23 @@ public void start(@NonNull final MessageFrame frame, @NonNull final OperationTra
             return;
         }
 
+        // Handle evm precompiles
```

Comment:
Question - why did this block have to move.  If there is a good explanation maybe include a comment?

Reply:
I moved it because of readability and changing operation order was important. 
But since then for consolidating the work with the MR you mentioned above ... I changed back the order. Thank you.